app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+
+  def self.recent(user)
+    Post.where(author: user).limit(3).order(created_at: :desc)
+  end
+
+  attr_accessor :bio, :name, :photo, :posts_counter
```

Comment:
Remove the following `attr_accessor` with its arguments because we don't need to generate the getters or setters methods for each given attribute.